fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/client/model/FabricModelProviderHelper.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.object.builder.v1.client.model;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.CrossbowItem;
+import net.minecraft.item.ElytraItem;
+import net.minecraft.item.FishingRodItem;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+@Environment(EnvType.CLIENT)
+public final class FabricModelProviderHelper {
```

Comment:
I really don't think we need this class, I think if we really want to help new developers the solution is not to give them easy "builder" for everything, but to push them in the right direction to get a better understanding of what's actually happening. The three ways that come to mind are by making better javadocs, writing better testmods, and writing wiki pages for this. If you think this needs more explanation I suggest moving an example of registering a model provider into the test mod and then writing a wiki page about model providers, not bloating fabric with what are really simple "builders" effectively.

Reply:
alright, i just feel that its helpful to have to not think about things like this, but better tutorials do make sense. Ill be implementing these in this module anyway, so those can be the reference.